### PR TITLE
Update Permissions

### PR DESCRIPTION
**Pull request by Lulalaby**
See https://github.com/discord/discord-api-docs/pull/3128/files & https://discord.com/developers/docs/topics/permissions

### CODE REVIEW

**Comment by Lulalaby** [comment]
`MANAGE_EVENTS` is not in the docs atm but is working already.
I added it to fix the gap between `REQUEST_TO_SPEAK` & `MANAGE_THREADS`.

File: src/main/java/com/mewna/catnip/entity/util/Permission.java [hunk 1]
```diff
@@ -70,8 +70,13 @@
...
     MANAGE_ROLES(1L << 28, true, "Manage Roles"),
     MANAGE_WEBHOOKS(1L << 29, true, "Manage Webhooks"),
-    MANAGE_EMOJIS(1L << 30, false, "Manage Emojis"),
+    MANAGE_EMOJIS_AND_STICKERS(1L << 30, false, "Manage Emojis & Stickers"),
     USE_APPLICATION_COMMANDS(1L << 31, false, "Use Slash Commands"),
+    REQUEST_TO_SPEAK(1L << 32, false, "Request to Speak in Stage Channels"),
+    MANAGE_EVENTS(1L << 32, false, "Manage Guild Events"), // This permission is still an experiment, but will come soon
+    MANAGE_THREADS(1L << 34, false, "Manage Threads"),
+    USE_PUBLIC_THREAD(1L << 35, false, "Use Public Threads"),
+    USE_PRIVATE_THREADS(1L << 36, false, "Use Private Threads"),
```
Comment (queer) [inline src/main/java/com/mewna/catnip/entity/util/Permission.java:79]: These are included as a part of #541 and do not need to be here.

Comment (Lulalaby) [reply src/main/java/com/mewna/catnip/entity/util/Permission.java:79]: ok

Comment (Lulalaby) [reply src/main/java/com/mewna/catnip/entity/util/Permission.java]: fixed

**Comment by Lulalaby** [comment]
@queer I am sorry, I accidentally removed slash commands

**Pull request merged**